Modules/@babylonjs/react-native-basekit/scripts/install.js

```diff
@@ -0,0 +1,227 @@
+const https = require("https");
+const zlib = require("zlib");
+const tar = require("tar");
+const path = require("path");
+const fs = require("fs");
+
+function getCachePathCandidates() {
+  return [
+    process.env.npm_config_babylon_binary_cache,
+    process.env.npm_config_cache,
+  ].filter(function(_) {
+    return _;
+  });
+}
+
+function getCached(package, binaryFilename) {
+  return [
+    getCachePathCandidates(),
+    "BabylonReactNative",
+    package.version,
+    binaryFilename,
+  ].join("/");
+}
+
+function useCachedFile(filePath) {
+  return new Promise((resolve, reject) => {
+    if (fs.existsSync(filePath)) {
+      const gunzip = zlib.createGunzip();
+      const untar = tar.extract({ cwd: process.cwd() }); // Extract to the current working directory
+
+      const sourceStream = fs.createReadStream(filePath);
+      sourceStream
+        .pipe(gunzip) // Decompress the .tar.gz file
+        .pipe(untar) // Extract the tar contents to the cwd
+        .on("error", (err) => {
+          reject(err);
+        })
+        .on("end", () => {
+          resolve(true);
+        });
+    } else {
+      resolve(false);
+    }
+  });
+}
+
+function getArgument(name) {
+  const flags = process.argv.slice(2),
+    index = flags.lastIndexOf(name);
```

Comment:
This seems unusual, maybe make it more standard?
```suggestion
  const flags = process.argv.slice(2);
  const index = flags.lastIndexOf(name);
```

Reply:
Fixed